trellis-triplestore/src/main/java/org/trellisldp/triplestore/TriplestoreResourceService.java

```diff
@@ -141,58 +140,71 @@ private static RDFTerm getBaseIRI(final RDFTerm object) {
     }
 
     @Override
-    public Future<Boolean> put(final IRI identifier, final IRI ixnModel, final Dataset dataset) {
+    public Future<Boolean> create(final IRI identifier, final IRI ixnModel, final Dataset dataset) {
+        LOGGER.debug("Creating: {}", identifier);
+        return supplyAsync(() -> createOrReplace(identifier, ixnModel, dataset, OperationType.CREATE));
+    }
+
+    @Override
+    public Future<Boolean> delete(final IRI identifier, final IRI ixnModel, final Dataset dataset) {
+        LOGGER.debug("Deleting: {}", identifier);
         return supplyAsync(() -> {
-            final Boolean isDelete = dataset.contains(of(PreferAudit), null, RDF.type, AS.Delete);
             final Instant eventTime = now();
+            dataset.clear();
```

Comment:
Actually, this is a bug, and I take credit for introducing it. Simply removing this line should work. See related conversation here: https://github.com/trellis-ldp/trellis/pull/31#discussion_r169131736

Reply:
You want me to pull out that line in this PR?

Reply:
Yes, please. It would have been in one of the commits I added to the PR. Alternately, we can let it stay and I'll open a separate issue to pull it out after the PR is merged.

Reply:
Done.